src/main/java/DrukmakoriSivatag/Random.java

```diff
@@ -9,73 +8,77 @@
 public class Random {
 
     /**
-     *  Egy random generált érték alapján eldönti, hogy elromoljon-e a pumpa, vagy sem, majd visszatér a döntéssel
+     * Random object for generating random numbers
+     */
+    private final java.util.Random myRandom = new java.util.Random();
```

Comment:
Using a pseudorandom generator is not safe, try using another random generator instead.